appformer-ala-wildfly-provider/src/main/java/org/kie/appformer/ala/wildfly/executor/AppFormerWildflyRuntimeExecExecutor.java

```diff
@@ -100,129 +101,176 @@ public AppFormerWildflyRuntimeExecExecutor( final RuntimeRegistry runtimeRegistr
                 }
             }
         }
-        Optional< WildflyRuntime > result = super.apply( ( WildflyRuntimeConfiguration ) runtimeConfig );
-        logger.info( "Executor finished, wildflyRuntime: " + result.get( ) );
+        Optional<WildflyRuntime> result = super.apply((WildflyRuntimeConfiguration) runtimeConfig);
+        logger.info("Executor finished, wildflyRuntime: " + result.get());
         return result;
     }
 
-    private void prepareForProvisioning( AppFormerWildflyRuntimeExecConfig runtimeConfig, WildflyProvider wildflyProvider ) {
-        Path warPath = Paths.get( runtimeConfig.getWarPath( ) );
-        String warName = warPath.getFileName( ).toString( );
+    private void prepareForProvisioning(AppFormerWildflyRuntimeExecConfig runtimeConfig,
+                                        WildflyProvider wildflyProvider) {
+        Path warPath = Paths.get(runtimeConfig.getWarPath());
+        String warName = warPath.getFileName().toString();
 
-        WildflyAppState appState = wildfly.getWildflyClient( wildflyProvider ).getAppState( warName );
-        if ( "Running".equals( appState.getState( ) ) && runtimeConfig.getRedeployStrategy( ).equals( "auto" ) ) {
+        WildflyAppState appState = wildfly.getWildflyClient(wildflyProvider).getAppState(warName);
+        if ("Running".equals(appState.getState()) && runtimeConfig.getRedeployStrategy().equals("auto")) {
```

Comment:
sorrry, here, in the new code I've meant org.guvnor.ala.runtime.RuntimeState.RUNNING